### PR TITLE
Color removal

### PR DESCRIPTION
**Pull request by JacquesTa**
Fixes https://github.com/serlo/overview.serlo.org/issues/81

### CODE REVIEW

File: template.html [hunk 1]
```diff
@@ -125,13 +114,10 @@ <h1 id="{{name|urlencode}}">{{name}}</h1>
           <aside class="legend">
```
Comment (kulla) [inline template.html:114]: Here you can delete the whole `<aside class="legend">...</aside>` tag...

**Comment by JacquesTa** [comment]
New commit #81

**Comment by kulla** [comment]
You can also remove https://github.com/serlo/overview.serlo.org/blob/master/serlo/model.py#L179-L193

**Comment by JacquesTa** [comment]
#81 new commit

**Pull request merged**